packages/assets-controllers/src/token-prices-service/codefi-v2.test.ts

```diff
@@ -1,12 +1,15 @@
 import nock from 'nock';
+import { useFakeTimers } from 'sinon';
 
 import {
-  codefiTokenPricesServiceV2,
+  CodefiTokenPricesServiceV2,
   SUPPORTED_CHAIN_IDS,
   SUPPORTED_CURRENCIES,
 } from './codefi-v2';
 
-describe('codefiTokenPricesServiceV2', () => {
+const defaultMaxRetryDelay = 30_000;
```

Comment:
This number comes from here, correct? https://github.com/connor4312/cockatiel?tab=readme-ov-file#exponentialbackoff

Reply:
Yep, exactly. I'll add it in a comment

```suggestion
// We're not customizing the default max delay
// The default can be found here: https://github.com/connor4312/cockatiel?tab=readme-ov-file#exponentialbackoff
const defaultMaxRetryDelay = 30_000;
```